modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -91,6 +93,18 @@ def veteran_from_headers(with_gender: false)
         vet
       end
 
+      def validate_ccg_token!
+        client = TokenValidation::V2::Client.new(api_key: Settings.token_validation.api_key)
+        request_method_to_scope = {
+          'GET' => 'claim.read',
+          'PUT' => 'claim.write',
+          'POST' => 'claim.write'
+        }
```

Comment:
to my knowledge, these are the only 2 OAuth scopes for the claims endpoints. Since `token_valid` currently checks against a single scope, I figured this is a good way to generalize what scopes different request types get